### PR TITLE
Main README refactoring

### PR DESCRIPTION
**Pull request by barroco**
This PR simplifies the main README and clarifies the content of the repository for visitors which may be interested in monitoring modules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,44 @@
...
-Serves as an HTTPS gateway to the business logic, translating between
-HTTPS request and gRPC to allow users to communicate to the DSS via
-simple HTTPS calls. This code is currently generated via
-[grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway), and does
-not do much other than translation.  See the [API specification
-here](https://tiny.cc/dssapi_rid).
+## Discovery and Synchronization Service (DSS)
 
-### gRPC backend
+The DSS is a simple and open service used by separate USSs, often in different organizations, to communicate information about UAS operations and coordinate with each other. This service is described in the ASTM remote ID standard. This flexible and distributed system is used to connect multiple USSs operating in the same general area to share information while protecting operator and consumer privacy. The system is focused on facilitating communication amongst actively operating USSs without details about UAS operations stored or processed in the DSS.
```
Comment (BenjaminPelletier) [inline README.md:18]: nit: Can this line be easily broken into multiple lines?  It makes future diffs easier to view (same comment for other lines beyond ~110 characters)

File: README.md [hunk 2]
```diff
@@ -1,72 +1,44 @@
...
+- [Building and deploying a DSS instance](./build/README.md)
+- [Conceptual background on the DSS and services](./concepts.md)
+- [DSS implementation details](./implementation_details.md)
 
-### CockroachDB (CRDB)
+## Monitoring and UAS Service Suppliers (USS) testing
 
-Individual CockroachDB nodes hosting sharded data of the DAR. More information about CockroachDB
-[here](https://www.cockroachlabs.com/docs/stable/architecture/overview.html).
+In addition to the DSS, this repository contains tools for USSs to test and validate their implementation of the services such as Remote ID (ASTM F3411) and Strategic Conflict Detection defined in ASTM WK63418, UAS Traffic Management (UTM) UAS Service Supplier (USS) Interoperability Specification.
```
Comment (BenjaminPelletier) [inline README.md:27]: F3411-19

**Review by BenjaminPelletier — COMMENTED** [review]
This looks good -- just a few small comments

**Comment by barroco** [comment]
Thank you @BenjaminPelletier for the review, comments should be resolved.

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**